tests/Microsoft.Bot.Builder.Ai.Translation.Tests/TranslatorTests.cs

```diff
@@ -69,18 +74,111 @@ public class TranslatorTests
             }
 
             Translator translator = new Translator(translatorKey);
-            translator.SetPostProcessorTemplate(new List<string> { "perr[oa]" });
+            Dictionary<string, List<string>> patterns = new Dictionary<string, List<string>>();
```

Comment:
This entire test needs to be broken down into a larger number of simpler tests. Those tests need to focus on boundry condidtions, passing in null variables, passing in empty collections, and the like. Once *those* tests pass, the actual functionality needs to be tested.